### PR TITLE
add docs for shards' capacity health indicator

### PR DESCRIPTION
**Pull request by HiDAl**

### CODE REVIEW

**Comment by HiDAl** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by HiDAl** [comment]
@elasticmachine update branch

File: docs/reference/health/health.asciidoc [hunk 1]
```diff
@@ -377,6 +381,36 @@ watermark threshold>>.
     The `invocations_since_last_success` key will report a map where the unhealthy policy
     name is the key and it's corresponding number of failed invocations is the value.
 
+[[health-api-response-details-shards-usage]]
+===== shards_capacity
+`data`::
+(map) A view with information about the current capacity of shards for data nodes that do not belong to the frozen layer.
```
Comment (andreidan) [inline docs/reference/health/health.asciidoc:387]:
```suggestion
(map) A view with information about the current capacity of shards for data nodes that do not belong to the frozen tier.
```

File: docs/reference/health/health.asciidoc [hunk 2]
```diff
@@ -377,6 +381,36 @@ watermark threshold>>.
...
+`max_shards_in_cluster`::
+(int) Indicates the maximum number of shards that the cluster can hold.
+
+`current_used_shards`::
+(optional, int) The total number of shards hold by the cluster. Only displayed in the case the indicator's status is `red` or `yellow`.
+
+=====
+
+`frozen`::
+(map) A view with information about the current capacity of shards for data nodes that belong to the frozen layer.
```
Comment (andreidan) [inline docs/reference/health/health.asciidoc:401]:
```suggestion
(map) A view with information about the current capacity of shards for data nodes that belong to the frozen tier.
```

File: docs/reference/health/health.asciidoc [hunk 3]
```diff
@@ -377,6 +381,36 @@ watermark threshold>>.
...
++
+.Properties of `frozen`
+[%collapsible%open]
+=====
+`max_shards_in_cluster`::
+(int) Indicates the maximum number of shards that the cluster can hold.
```
Comment (andreidan) [inline docs/reference/health/health.asciidoc:407]: Should we say these are the frozen shards?

Comment (andreidan) [inline docs/reference/health/health.asciidoc:407]:
```suggestion
(int) Indicates the maximum number of shards the cluster can hold for the partially mounted indices.
```

**Review by andreidan — APPROVED** [review]
LGTM, thanks Pablo

**Pull request merged**